Production/nginx.tmpl

```diff
@@ -184,6 +208,31 @@ map $http_upgrade $proxy_connection {
   '' close;
 }
 
+{{ if (eq $serviceName "lnd_lit") }}
+# Redirect all incoming grpc-web requests to the LiT proxy. Everything else
+# goes directly to the internal lnd container. We use a temporary variable
+# here so we can rewrite it to the actual host again in the next map
+# directive.
+map $content_type $tmp_backend {
+  "application/grpc-web+proto" "lnd_lit:8080"; # LiT proxy.
```

Comment:
Because this variable decides whether the request goes to `LiT` or `lnd`, you can't mix HTTP and HTTPS. So I think this needs to be `lnd_lit:8443` so it's always TLS. Same below.